renderer/components/ui/AutocompleteSelector.tsx

```diff
@@ -89,13 +89,14 @@ export default function AutocompleteSelector({
   };
 
   const onChangeHandler = (e: ChangeEvent<HTMLInputElement>) => {
+    setSelectedItem(e.target.value);
+  };
+
+  const onBlurHandler = (e: ChangeEvent<HTMLInputElement>) => {
     if (e.target.value.startsWith(" ")) {
       e.target.value = e.target.value.trim();
     }
```

Comment:
i guess you should trim even you have spaces at the end
so i would remove condition
leave just
`e.target.value = e.target.value.trim();`